keel-plugin/src/main/kotlin/com/netflix/spinnaker/keel/plugin/TaskLauncher.kt

```diff
@@ -75,7 +82,31 @@ class TaskLauncher(
     subject: String,
     description: String,
     correlationId: String,
-    stages: List<Map<String, Any?>>
+    stages: List<Map<String, Any?>>,
+    artifacts: List<Artifact> = emptyList()
+  ): Task =
+    submitJobToOrca(
+      serviceAccount = serviceAccount,
+      application = application,
+      notifications = notifications,
+      subject = subject,
+      description = description,
+      correlationId = correlationId,
+      stages = stages,
+      type = SubjectType.CONSTRAIN,
```

Comment:
@emjburns following our conversation, I added a `type` property. Since I didn't want to change the signature of the current calls, I added another overloading method. Let me know what you think!